hibernate-reactive-core/src/main/java/org/hibernate/reactive/mutiny/impl/MutinySessionFactoryImpl.java

```diff
@@ -171,6 +185,23 @@ private ReactiveConnection proxyConnection(String tenantId) {
 				} );
 	}
 
+	@Override
+	public <T> Uni<T> withStatelessSession(String tenantId, Function<Mutiny.StatelessSession, Uni<T>> work) {
+		String id = statelessSessionId();
```

Comment:
@blafond this would need to be an id that includes the tenant id. At least that's how I did it with regular sessions.

Reply:
ah.. I missed that. thx

Reply:
@DavideD Note locally this [test](https://github.com/hibernate/hibernate-reactive/blob/da7c79cf87ca88942b53bcdd15040c9253eb89c3/hibernate-reactive-core/src/test/java/org/hibernate/reactive/ReactiveMultitenantNoResolverTest.java#L304) is failing

Though a HibernateException is getting thrown as expected.  The Matcher isn't matching because the expected exception isn't wrapped in a CompletionException like in the previous test in that class.  Can't figure out why though, but guessing it's in my `withStatelessSession(String tenantId, ....)` logic which I mimic'd from the non-tenantId version.

Reply:
@blafond if this is testing the _mutiny_ API then I don’t see why a `ConpletionException` would be correct.